src/docs/doxygen/Doxyfile.in

```diff
@@ -788,7 +788,7 @@ INPUT                  = @PROJECT_SOURCE_DIR@/axom/doxygen_mainpage.md \
                          @PROJECT_SOURCE_DIR@/axom/mint/mesh \
                          @PROJECT_SOURCE_DIR@/axom/mint/utils \
                          @PROJECT_SOURCE_DIR@/axom/mint/execution \
-                         @PROJECT_SOURCE_DIR@/axom/primal/README.md \
+                         @PROJECT_SOURCE_DIR@/axom/primal/doxygen_mainpage.md \
```

Comment:
This change wires the primal doxygen back in.

Reply:
If you rebase on develop it has this change already from my PR that i just merged yesterday. Stop taking credit for my work arlie! I'm onto you! =)

https://github.com/LLNL/axom/commit/17af1877ed7a88757168e9efe74d3a93f14c3724#diff-39217672349aed9d8313ef4107df61bd

Reply:
Quack!!  ---Thanks for fixing that Chris.